Test/Service/ServiceTest/AbstractServiceTest.cs

```diff
@@ -43,6 +45,7 @@ public AbstractServiceTest()
         protected static SessionDecorator Session { get; private set; }
         protected IUnitOfWork UnitOfWork;
         protected IUser User;
+        protected IOrganization Org;

```

Comment:
For the most part, developers have stopped abbreviating except for some very common abbreviations. You might consider spelling out the word.